openshift_cli_installer/libs/clusters/ipi_cluster.py

```diff
@@ -181,3 +182,75 @@ def destroy_cluster(self):
         self.run_installer_command(action=DESTROY_STR, raise_on_failure=True)
         self.logger.success(f"{self.log_prefix}: Cluster destroyed")
         self.delete_cluster_s3_buckets()
+
+
+class AwsIpiCluster(IpiCluster):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.logger = get_logger(f"{self.__class__.__module__}-{self.__class__.__name__}")
+
+        self.platform = AWS_STR
+        if self.create:
+            self._create_install_config_file(platform=self.platform)
+        self.dump_cluster_data_to_file()
```

Comment:
should be moved to `IpiCluster`
create to `super` last in the init (after setting all specific values)